doc/tasks.md

```diff
@@ -129,6 +130,7 @@ Every task has its own default configuration. It is possible to overwrite the pa
 - [Stylelint](tasks/stylelint.md)
 - [Tester](tasks/tester.md)
 - [TwigCs](tasks/twigcs.md)
+- [TwigCsFixer](tasks/twigcsfixer.md)
```

Comment:
```suggestion
- [Twig-CS-Fixer](tasks/twigcsfixer.md)
```

Reply:
I don't know why I thought all the names used Camel Case, but checking it now there are several with spaces in the name. Do you prefer the name with hyphen or spaces?

Reply:
I created my tool with PHP-CS-Fixer as inspiration, so let's go with `Twig-CS-Fixer` since `PHP-CS-Fixer` is written like this in these tasks.

Reply:
Perfect!